homeassistant/components/asuswrt/__init__.py

```diff
@@ -81,7 +81,7 @@ async def async_setup(hass, config):
     options = {}
     mode = conf.get(CONF_MODE, MODE_ROUTER)
     for name, value in conf.items():
-        if name in ([CONF_DNSMASQ, CONF_INTERFACE, CONF_REQUIRE_IP]):
+        if name in (CONF_DNSMASQ, CONF_INTERFACE, CONF_REQUIRE_IP):
```

Comment:
Should we not be consistent? There are tuples ending with `,`  and not

Reply:
This is somewhat special. The parentheses here weren't used before, i.e.
```py
([CONF_DNSMASQ, CONF_INTERFACE, CONF_REQUIRE_IP])  == [CONF_DNSMASQ, CONF_INTERFACE, CONF_REQUIRE_IP]
```
There wasn't a trailing comma, so no tuple.

Take a look at the next line as well: `if name == CONF_REQUIRE_IP ...`. That suggests that it should only be a normal list / tuple. Not a tuple of tuples.

Reply:
Sure, I took this as an example (bad one), but it's still mixed and matched everywhere

Reply:
You normally don't need a trailing comma for tuples. Only if there is just one value.

I won't recommend adding additional once either as this would change the formatting with black.